integrations/slack/webhooks/webhook.go

```diff
@@ -195,3 +234,46 @@ func (h handler) dispatchAsyncEventsToConnections(ctx context.Context, cids []sd
 		l.Debug("Event dispatched")
 	}
 }
+
+func (h handler) extractIDs(body []byte, wantContentType string, l *zap.Logger) (appID, teamID, enterpriseID string, err error) {
+	if wantContentType == "application/json" {
```

Comment:
Check that the type **starts with** this string, not `==`, to support custom encoding suffixes